README.md

```diff
@@ -164,11 +164,11 @@ Similarly, one can effectively _cancel_ the execution of an iterator at any time
 Our iterator is nothing more than a vanilla structure (its state) with a `.next` method defined on it.  
 In this example, the iterator sits on `main`'s stack, and answers to the same rules as any other struct: ownership, lifetimes, marker traits, etc.
 
-There really isn't any kind of magic going on here: no hidden allocations, no codegen, no nothing; it's as dull as it gets.
+There really isn't any kind of magic going on here: no hidden allocations, no codegen, no nothing.  It's as dull as it gets.
```

Comment:
The colon implies that what follows elucidates what precedes it.  That's true only up to the semicolon.

https://painintheenglish.com/case/616